spec/avro_spec.rb

```diff
@@ -172,7 +269,7 @@
     }.to raise_error(AvroTurf::SchemaNotFoundError, "could not find Avro schema at `spec/schemas/not_there.avsc'")
   end
 
-  it "raises AvroTurf::SchemaError if the schema's namespace doesn't match the file location" do
+  it "raises AvroTurf::InvalidSchemaError if the schema's namespace doesn't match the file location" do
     FileUtils.mkdir_p("spec/schemas/test/people")
```

Comment:
Why is there `spec/schemas/test` when all of the schemas under `spec` are test specs?  That is, why use the word `test` when already using the word `spec`?